src/main/java/httpServer/App.java

```diff
@@ -1,7 +1,16 @@
 package httpServer;
 
+import java.net.ServerSocket;
+
 public class App {
-    public String hello() {
-        return "Hello world";
+    public static void main(String[] args) {
+        try {
+            var serverSocket = new ServerSocket(5000);
+            var executor = new CurrentThreadExecutor();
+            new Server(serverSocket, executor).start();
```

Comment:
This is personal preference, but I'd write this as:

```
var server = new Server(new ServerSocket(5000), new CurrentThreadExecutor());
server.start();
```

Reply:
Thinking about it - I think I like to split creation from action

Reply:
Thanks, I'll look into this